src/pybind/mgr/prometheus/module.py

```diff
@@ -1,6 +1,6 @@
 import cherrypy
 from collections import defaultdict
-from distutils.version import StrictVersion
+from packaging.version import Version
```

Comment:
@tchaikov this broke the prometheus module in some of our testing environment with the error..
```
Jan 10 06:52:01 ceph-node-00 ceph-mgr[13294]: mgr[py] Module not found: 'prometheus'
Jan 10 06:52:01 ceph-node-00 conmon[13288]: 2023-01-10T06:52:01.904+0000 7fa8c5ad5200 -1 mgr[py] Traceback (most recent call last):
Jan 10 06:52:01 ceph-node-00 conmon[13288]:   File "/usr/share/ceph/mgr/prometheus/__init__.py", line 2, in <module>
Jan 10 06:52:01 ceph-node-00 conmon[13288]:     from .module import Module, StandbyModule
Jan 10 06:52:01 ceph-node-00 conmon[13288]:   File "/usr/share/ceph/mgr/prometheus/module.py", line 3, in <module>
Jan 10 06:52:01 ceph-node-00 conmon[13288]:     from packaging.version import Version
Jan 10 06:52:01 ceph-node-00 conmon[13288]: ModuleNotFoundError: No module named 'packaging'
Jan 10 06:52:01 ceph-node-00 conmon[13288]: 
Jan 10 06:52:01 ceph-node-00 ceph-mgr[13294]: mgr[py] Traceback (most recent call last):
                                                File "/usr/share/ceph/mgr/prometheus/__init__.py", line 2, in <module>
                                                  from .module import Module, StandbyModule
                                                File "/usr/share/ceph/mgr/prometheus/module.py", line 3, in <module>
                                                  from packaging.version import Version
                                              ModuleNotFoundError: No module named 'packaging'
Jan 10 06:52:01 ceph-node-00 conmon[13288]: 2023-01-10T06:52:01.914+0000 7fa8c5ad5200 -1 mgr[py] Class not found in module 'prometheus'
```

So my doubt is since we introduced the `packaging` to the mgr module, don't we need to add it to the mgr dependencies? otherwise this will be broken right?

Reply:
@nizamial09, https://github.com/ceph/ceph/pull/49579

Reply:
ah yes! thank you @Matan-B